parla/ldevice.py

```diff
@@ -126,8 +127,8 @@ def partition(self, data, memory_kind: MemoryKind = None):
         :return: A Python list of objects returned by `data` and copied to the appropriate device.
         """
```

Comment:
You have changed the semantics of these functions to return special objects that do lazy copies (they are not quite normal views, but that's fine). Update the doc strings to reflect the changes. You may well have to update the docs in many other functions in this file. You can search for "copied to the appropriate device" since that's the part the changed more than anything. 

I know we are really tight on time. I would be happy with adding a TODO to each doc string that will need updating saying `Docs need updating. The semantics have changed. See PartitionedNDArray.`